lighthouse-core/test/audits/metrics/cumulative-long-queuing-delay-test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../audits/metrics/cumulative-long-queuing-delay.js');
+const options = Audit.defaultOptions;
+
+const pwaTrace = require('../../fixtures/traces/progressive-app-m60.json');
+
+function generateArtifactsWithTrace(trace) {
+  return {
+    traces: {[Audit.DEFAULT_PASS]: trace},
+    devtoolsLogs: {[Audit.DEFAULT_PASS]: []},
+  };
+}
+/* eslint-env jest */
+
+describe('Performance: cumulative-long-queuing-delay audit', () => {
+  it('evaluates cumulative long queuing delay metric properly', async () => {
+    const artifacts = generateArtifactsWithTrace(pwaTrace);
+    const settings = {throttlingMethod: 'provided'};
+    const context = {options, settings, computedCache: new Map()};
+    const output = await Audit.audit(artifacts, context);
+
+    expect(output.numericValue).toBeCloseTo(48.3, 1);
+    expect(output.score).toBeCloseTo(1, 2);
```

Comment:
maybe add a comment here that it's expected to be `// very nearly 1` or something